pkg/docker/manifest.json.in

```diff
@@ -6,7 +6,7 @@
 
     "menu": {
         "index": {
-            "label": "Docker Containers",
+            "label": "Docker containers",
```

Comment:
PF4 says "Write menu/navigation items and their corresponding page titles in title case." @ https://www.patternfly.org/v4/design-guidelines/content/capitalization#capitalization-across-products — I'm not sure if I agree with that. Let's see what it looks like.

(We may want to go over this PR with this in mind if we do want to have navigation remain Title Case.)